src/main/java/io/pravega/connectors/flink/EventTimeOrderingOperator.java

```diff
@@ -85,6 +85,12 @@ public void setInputType(TypeInformation<?> type, ExecutionConfig executionConfi
     @Override
     public void initializeState(StateInitializationContext context) throws Exception {
         super.initializeState(context);
```

Comment:
As it is only super method inside, we can delete this `initializeState` function.